packages/svelte2tsx/test/htmlx2jsx/index.ts

```diff
@@ -1,6 +1,12 @@
 import { htmlx2jsx } from '../build/htmlxtojsx';
 import { test_samples } from '../helpers';
 
-describe('htmlx2jsx', () => {
-    test_samples(__dirname, htmlx2jsx, 'jsx');
+describe('htmlx2jsx', function () {
```

Comment:
why the change to `function` (here and below)?

Reply:
I took the habit to avoid inline functions when possible (worse performance), I should avoid doing that in PRs